app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.find_by_email(params[:email])
+    if user
+      user.generate_token
+      AppMailer.password_reset_link(user).deliver
+      render :confirm
```

Comment:
Thanks. 'will try to remember this.
I googled to read more and found [this](http://www.theserverside.com/news/1365146/Redirect-After-Post).